spec/event_spec.rb

```diff
@@ -0,0 +1,45 @@
+require './lib/person'
+require './lib/craft'
+require './lib/event'
```

Comment:
I like that you are using additional expectations for things like Arrays but make sure you are also including the actual expectations that are given in the interaction patterns. Ie
```
pry(main)> event.crafts
#=> [#<Craft:0x00007fd37b818318...>]

pry(main)> event.attendees
#=> [#<Person:0x00007fd379b96028...>]
```